Providers/Scripts/2.4x-2.5x/Scripts/Tests/test_nxOMSAutomationWorker.py

```diff
@@ -228,28 +228,29 @@ def test_get_stray_worker_and_manager_pids(self):
         stray_processes = nxOMSAutomationWorker.get_stray_worker_and_manager_wsids(processes, self.workspace_id)
         self.assertFalse(stray_processes)
 
-    def test_is_any_1_3_process_running(self):
+    def test_is_any_1_4_process_running(self):
         processes = [
-            '1002 python %s /var/opt/microsoft/omsagent/state/automationworker/oms.conf rworkspace:cfd4ef08-4011-428a-8947-0c2f4605980f 1.3' % nxOMSAutomationWorker.WORKER_MANAGER_START_PATH,
-            '1003 python %s /var/opt/microsoft/omsagent/state/automationworker/oms.conf rworkspace:cfd4ef08-4011-428a-8947-0c2f4605980f 1.4' % nxOMSAutomationWorker.WORKER_MANAGER_START_PATH,
+            '1002 python %s /var/opt/microsoft/omsagent/state/automationworker/oms.conf rworkspace:cfd4ef08-4011-428a-8947-0c2f4605980f 1.4' % nxOMSAutomationWorker.WORKER_MANAGER_START_PATH,
+            '1003 python %s /var/opt/microsoft/omsagent/state/automationworker/oms.conf rworkspace:cfd4ef08-4011-428a-8947-0c2f4605980f 1.5.0.0' % nxOMSAutomationWorker.WORKER_MANAGER_START_PATH,
             '']
-        self.assertTrue(nxOMSAutomationWorker.is_any_1_3_process_running(processes, self.workspace_id))
+        self.assertTrue(nxOMSAutomationWorker.is_any_1_4_process_running(processes, self.workspace_id))
```

Comment:
Doesn't have to be done now but can you also assert that only 1 processis found here.